### ISSUE TITLE
[HELP]: How to parse FoodData.buffer

### ISSUE THREAD

**Issue by Yi-Jiahe** [Canardoux/flutter_sound#844]
Following the 'recordToStream/recordToStreamExample.dart' example, I listen to my StreamController<Food> and set the ByteBuffer content of the FoodData to the state for processing.
```
var recordingDataController = StreamController<Food>();
    _mRecordingDataSubscription =
        recordingDataController.stream.listen((buffer) {
      if (buffer is FoodData) {
        setState(() {
          _data = buffer.data!.buffer;
        });
        sink.add(buffer.data!);
      }
    });
    await _mRecorder!.startRecorder(
      toStream: recordingDataController.sink,
      codec: Codec.pcm16,
      numChannels: 1,
      sampleRate: tSampleRate,
    );
    setState(() {});
```
As it is a pcm16 buffer, I process it by taking pairs of bytes and reading them as signed 16 bit integers
```
int nSamples = data.lengthInBytes ~/ 2;
for (var _i =   0; _i < nSamples; _i++) {
      var sample = data.asByteData(_i*2, 2).getInt16(0) / 32768;

    /* Do something with samples */
}
```

However I have found that there at ~70 bytes at the start of each buffer which are the same and another ~1? byte after that which is non-zero after the screen goes black for a while and all the values are 0 (which is another issue I am having; is there some way to keep receiving valid microphone data when the phone is sleeping?). I was looking for some documentation regarding this header? but I couldn't find anything on how to process it and how long it is.

I did notice someone had a similar issue in issue #507 so I was wondering if perhaps it is a implementation thing and I should just offset the data?

**Comment by Larpoux** [comment]
No, this is not normal.
This is perhaps a bug in Flutter Sound.
Is it on ios or Android?
I already realized that record to raw pcm on ios give some audio artifact. Many audible cliks.
I have fighted during several days, reading the code and re-reading it. But never found the reason.
Actually I have begun to work on Taudio. See [#118](https://https://github.com/Canardoux/flutter_sound/issues/118) 
I am not very motivated to do Flutter Sound maintenance.
I really would like that someone else do this maintenance.
